setup.py

```diff
@@ -0,0 +1,21 @@
+from setuptools import setup, find_packages
+
+with open('requirements.txt', 'r') as f:
+    install_requires = f.readlines()
+
+setup(name='pommerman',
+      version='0.1.0',
+      description='PlayGround: AI Research into Multi-Agent Learning',
+      url='https://www.pommerman.com',
+      author='Pommerman',
+      author_email='support@pommerman.com',
+      license='Apache 2.0',
+      packages=find_packages(),
+      install_requires=install_requires,
+      entry_points={
+        'console_scripts': [
+            'pom_battle=pommerman.cli.run_battle:main',
+            'pom_tf_battle=pommerman.cli.train_with_tensorforce:main',
```

Comment:
what's the reasoning behind having the train_with_tensorforce as a cli?

Reply:
It was only to keep the earlier functionality. I don't intend to keep it in the next revision. Perhaps this could go as another example of creating an agent with `tensorforce` and then wrapping it in a Docker image, which is another standard process for using this environment.